src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: $cardWidth;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-weight: 400px;
+  color: $lightTextColor;
+
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 3px 40px ;
+  }
+
+  &__productName {
+    font-weight: 500;
+    font-size: 1.2em;
+    line-height: 18px;
+    color: $darkTextColor;
+
+    margin-bottom: 4px;
+  }
+
+  &__productCode {
+    margin-bottom: 16px;
+
+    line-height: 14px;
+  }
+
+  &__product-rating {
+    @extend %centringContent;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $darkTextColor;
+  }
+
+  &__price {
+    @extend %centringContent;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 1.2em;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 1.6em;
+    line-height: 18px;
+    font-weight: 700;
+    color: $darkTextColor;
+  }
+
+  &__buy-button {
+    @extend %centringContent;
+
+    background-color: $styleColorMoyo;
+    border: 1px solid $styleColorMoyo;
+    border-radius: 5px;
+
+    padding: 10px 68px 11px;
+
+    color: #fff;
+    text-decoration: none;
+    font-size: 1.4em;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    transition: color 0.3s ease, background-color 0.3s ease;
+  }
```

Comment:
In this case, will the transition apply to all styles? Maybe the mentor will tell me how to do it better because in the previous task, another student advised me to write specific styles

Reply:
yeah, it will work correctly, I have checked it already, but I agree with you, it is better to get the right answer from the mentor :)

Reply:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated

Reply:
@ArtemHulkoDev it works, but better not to do it 🙂 It's bad for page performance